OrbitCaptureClient/include/OrbitCaptureClient/CaptureClient.h

```diff
@@ -56,7 +54,8 @@ class CaptureClient {
   }
 
  private:
-  void Capture(int32_t process_id, std::string process_name, std::shared_ptr<Process> process,
+  void Capture(std::unique_ptr<ProcessData> process,
+               absl::flat_hash_map<std::string, ModuleData*>&& module_map,
```

Comment:
Why? I like it this way, because it signals that the caller should provide a object that he resigns ownership of. I am happy to change it, but I don't really see a downside.

Reply:
I bet there is a reason to make it call-by-value. Would you mind explaining?

Reply:
First argument would be why to explicitly disallow copies on the call-side? E.g. for tests it might be useful to use that module map as expectation.
Otherwise, my C++ is a bit too limited. I had that discussion once with @beckerhe and he basically introduced that pattern to me (which is on a data sink, have value params and move on the caller side).

Reply:
As discussed offline. Both should be fine.

Reply:
As discussed, I decided to leave it like this